content/recover.md

```diff
@@ -0,0 +1,252 @@
+!!! note
+    Requires director v277.4.0 and CLI v7.3.0
+
+BOSH provides the `create-recovery-plan` and `recover` CLI commands to repair IaaS resources used by a specific deployment. The underlying machinery is very similar to [Cloud Check](cck.md), with several exceptions:
+
+1. There is a 2-step process: `create-recovery-plan` scans the deployment for problems and then prompts the user to generate a recovery plan, which is saved to a file.  The `recover` consumes that file.
+1. Resolutions to problems are selected by instance group and problem type. The `cloud-check` command asks for a resolution for each particular problem.
+1. When generating a recovery plan, `max_in_flight` can be overriden per instance group.  This can be handy to speed deployment recovery.  The `cloud-check` command uses the `max_in_flight` values in the deployment manfiest.
+
+
+Otherwise, the types of problems and the mechanism by which they are repaired are the same as in the `cloud-check` command.
+
+
+## Creating a recovery plan {: #create-recovery-plan }
+
+To create a recovery plan, invoke the  `bosh create-recovery-plan` like:
+
+```shell
+bosh create-recovery-plan recovery-plan.yml
+```
+
+```text
+Using environment '10.0.0.23' as client 'ops_manager'
+
+Using deployment 'problems'
+
+Task 223
+
+Task 223 | 17:17:43 | Scanning 9 VMs: Checking VM states (00:00:31)
+Task 223 | 17:18:14 | Scanning 9 VMs: 3 OK, 2 unresponsive, 4 missing, 0 unbound (00:00:00)
+Task 223 | 17:18:14 | Scanning 3 persistent disks: Looking for inactive disks (00:00:08)
+Task 223 | 17:18:22 | Scanning 3 persistent disks: 2 OK, 0 missing, 0 inactive, 1 mount-info mismatch (00:00:00)
+
+Task 223 Started  Tue Jul 11 17:17:43 UTC 2023
+Task 223 Finished Tue Jul 11 17:18:22 UTC 2023
+Task 223 Duration 00:00:39
+Task 223 done
+
+Instance Group 'cloud_controller_ng'
+
+
+Problem type: missing_vm
+
+#    Description
+241  VM for 'cloud_controller_ng/fb4db9d0-3225-49e0-95f0-02926718554f (2)' with cloud ID 'vm-2c2df389-1d6c-4373-64ca-82950edc5595' missing.
+
+1 missing_vm problems
+
+1: Skip for now
+2: Recreate VM without waiting for processes to start
+3: Recreate VM and wait for processes to start
+4: Delete VM reference
+missing_vm (1): 2
+
+Problem type: unresponsive_agent
+
+#    Description
+237  VM for 'cloud_controller_ng/6beb143e-055b-48ce-9eff-8236866b0dc7 (1)' with cloud ID 'vm-b457a975-73e0-4330-476c-86175eda1438' is not responding.
+
+1 unresponsive_agent problems
+
+1: Skip for now
+2: Reboot VM
+3: Recreate VM without waiting for processes to start
+4: Recreate VM and wait for processes to start
+5: Delete VM
+6: Delete VM reference (forceful; may need to manually delete VM from the Cloud to avoid IP conflicts)
+unresponsive_agent (1): 2
+
+Override current max_in_flight value of '50%'? [yN]:
```

Comment:
Might be nice to put either `n` or `<enter>` here to give the reader more of a clue as to what you did here. (Like, if I think about it for a minute, I can determine that you pressed enter, but it'd be nice to SEE that.)